src/schema/contentful/phoenix.js

```diff
@@ -76,6 +78,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
+    "The person's position within the organization"
```

Comment:
Ahh these are tricky to describe! How about we change this one to:
```suggestion
    "The person's relationship with the organization: member? employee?"
```